cli/src/args/mod.rs

```diff
@@ -30,15 +31,28 @@ const TCP_TRANSPORT: &str = "tcp";
 static CARGO_BIN_NAME: &str = env!("CARGO_BIN_NAME");
 static CARGO_PKG_HOMEPAGE: &str = env!("CARGO_PKG_HOMEPAGE");
 
+pub struct IggyMergedConsoleArgs {
```

Comment:
I see this is not used (yet?). Do I get it right that parameters (from contexts and given as options in cli) will be fetched from this struct?

Reply:
yes correct - I will implement this soon